src/list-items/list-item-label.html

```diff
@@ -60,10 +60,11 @@
         display: inline-block;
         padding: 3px 6px;
         border-radius: 6px;
-        color: var(--primary-background-color);
-        color: var(--tag-text-color);
-        background-color: var(--pr-status-color);
-        background-color: var(--tag-color);
+        color: var(--primary-background-color, --tag-text-color);
+        background-color: var(--pr-status-color, --tag-color);
+        overflow: hidden;
+        text-overflow: ellipsis;
+        max-width: 120%;
```

Comment:
Could you verify if this also works for you (it seemed to work in the devices overview)?
Otherwise I will change it back to 100%.